Samples/Samples/Converters/CheckBoxEventArgsConverter.cs

```diff
@@ -0,0 +1,23 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Globalization;
+using System.Text;
+using Xamarin.Forms;
+
+namespace Samples.Converters
```

Comment:
I'm not happy that we have to add these converters - we should be able to construct the UI to avoid these given this is a sample app. Want to avoid including too many additional files etc

Reply:
Removed EpochToDateTimeConverter in next PR request. will consider altering StartDatePickersEnabled to be within the setter, the current approach I took was to learn about best practice. But I understand since it's a sample we may want to consider reducing files down the line.